src/gql/mutations/restart-versions.graphql

```diff
@@ -0,0 +1,61 @@
+mutation RestartVersions(
+  $versionId: String!
+  $abort: Boolean!
+  $versionsToRestart: [VersionToRestart!]!
+) {
+  restartVersions(
+    versionId: $versionId
+    abort: $abort
+    versionsToRestart: $versionsToRestart
+  ) {
+    id
+    createTime
+    startTime
+    finishTime
+    revision
+    author
+    status
+    order
+    repo
+    project
+    activated
+    message
+    isPatch
+    taskCount
+    project
+    baseVersionID
+    projectIdentifier
+    versionTiming {
+      makespan
+      timeTaken
+    }
+    parameters {
+      key
+      value
+    }
+    manifest {
+      id
+      revision
+      project
+      branch
+      isBase
+      moduleOverrides
+      modules
+    }
+    patch {
+      id
+      patchNumber
+      alias
+      commitQueuePosition
+      canEnqueueToCommitQueue
+      childPatches {
+        baseVersionID
+        githash
+        id
+        projectIdentifier
+        taskCount
+        status
+      }
+    }
+  }
```

Comment:
Is all this information necessary? I don't think it is afaict.

Reply:
We should only fetch information that we think should have changed as a result in this mutation. Which is patch and child patch statuses, and task statuses